pyproject.toml

```diff
@@ -7,10 +7,12 @@ repository = "https://github.com/oskarsh/Yin-Yang"
 
 license = "MIT"
 readme = "README.md"
-packages = [{include = "yin_yang"}]
+packages = [
+  { include = "yin_yang" }
+]
 
 [tool.poetry.dependencies]
-python = "^3.10,<3.13"
+python = "^3.11,<3.13"
```

Comment:
3.11 is needed for some of the updated versions of deps. I just looked and it would cause trouble for people trying to build on Ubuntu 22.04, although funnily enough it doesn't cause problems on our Ubuntu 22.04 CI builds since we setup Python during the build. I suppose we have two options:

1. Ignore this since we will be pushing people towards the flatpak.
2. Downgrade some deps so we can build on 3.10.

I can look and see if there are any security vulnerabilities that were fixed in newer packages. If not, my vote would be to downgrade deps.